iam-bootstrap/render.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export deploy_id=example
```

Comment:
why do we have this file?

Reply:
A quick helper for anyone using the policies, but not provisioning them via the terraform  module.